Source/OxyPlot/Utilities/TrackerHelper.cs

```diff
@@ -0,0 +1,53 @@
+namespace OxyPlot.Utilities
+{
+    internal static class TrackerHelper
+    {
+        /// <summary>
+        /// Gets the nearest tracker hit.
+        /// </summary>
+        /// <param name="series">The series.</param>
+        /// <param name="point">The point.</param>
+        /// <param name="snap">Snap to points.</param>
+        /// <param name="pointsOnly">Check points only (no interpolation).</param>
+        /// <param name="firesDistance">The distance from the series at which the tracker fires</param>
+        /// <param name="isCheck"></param>
```

Comment:
Brief param comment needed here`: could the variable name be clearer (would `checkDistanceBetweenPoints` be objectionable?)

Reply:
Should mention that it is ignored if `pointsOnly` is set.